concordium-consensus/src/Concordium/ImportExport.hs

```diff
@@ -1,4 +1,5 @@
-{-# LANGUAGE MonoLocalBinds #-}
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
We don't usually want this. Why is it necessary?

Reply:
Fixed it.